src/main/java/com/fabriccommunity/spookytime/doomtree/DoomTree.java

```diff
@@ -0,0 +1,102 @@
+	public static Block PLACED_DOOM_LOG_CHANNEL = SpookyBlocks.register("doom_log_channel_p", new DoomLogBlock(logSettings().build(), true, 1));
+	public static Block PLACED_DOOM_LOG_TERMINAL = SpookyBlocks.register("doom_log_terminal_p", new DoomLogBlock(logSettings().build(), true, 1));
+
+	public static Block DOOM_LOG = Registry.register(Registry.BLOCK, SpookyTime.id("doom_log"),
+			new DoomLogBlock.Height(logSettings().dropsLike(PLACED_DOOM_LOG).build(), 0.04f));
+	public static Block DOOM_LOG_CHANNEL = Registry.register(Registry.BLOCK, SpookyTime.id("doom_log_channel"),
+			new DoomLogBlock.Height(logSettings().dropsLike(PLACED_DOOM_LOG_CHANNEL).build(), 0.02f));
+	public static Block DOOM_LOG_TERMINAL = Registry.register(Registry.BLOCK, SpookyTime.id("doom_log_terminal"),
+			new DoomLogBlock(logSettings().dropsLike(PLACED_DOOM_LOG_TERMINAL).build(), false, 0.02f));
+
+	public static Block DOOM_LEAF = SpookyBlocks.register("doom_leaves", new DoomLeafBlock(doomedSettings(), false, 1));
+
+	public static Block DOOMED_LOG = SpookyBlocks.register("doomed_log", new DoomedLogBlock(doomedSettings()));
+	public static Block DOOMED_EARTH = SpookyBlocks.register("doomed_earth", new DoomedBlock(doomedSettings()));
+	public static Block DOOMED_STONE = SpookyBlocks.register("doomed_stone", new DoomedBlock(doomedSettings()));
+	public static Block DOOMED_DUST = SpookyBlocks.register("doomed_dust", new DoomedBlock(doomedSettings()));
+	
+	public static Block DOOMED_RESIDUE_BLOCK = SpookyBlocks.register("doomed_residue_block", new Block(FabricBlockSettings.of(Material.EARTH).breakByHand(true).sounds(BlockSoundGroup.SAND).build()));
+	public static Block WARDING_ESSENCE_BLOCK = SpookyBlocks.register("warding_essence_block", new Block(FabricBlockSettings.of(Material.EARTH).breakByHand(true).sounds(BlockSoundGroup.SAND).build()));
+	
+	public static Item DOOMED_RESIDUE_ITEM = Registry.register(Registry.ITEM, SpookyTime.id("doomed_residue"), new Item((new Item.Settings()).group(SpookyTime.GROUP)));
+	public static Item WARDING_ESSENCE_ITEM = Registry.register(Registry.ITEM, SpookyTime.id("warding_essence"), new Item((new Item.Settings()).group(SpookyTime.GROUP)));
+	public static Item DOOM_FRAGMENT_ITEM = Registry.register(Registry.ITEM, SpookyTime.id("doom_fragment"), new Item((new Item.Settings()).group(SpookyTime.GROUP)));
+	public static Item ALCHEMICAL_BASIN_FRAME = Registry.register(Registry.ITEM, SpookyTime.id("alchemical_basin_frame"), new Item((new Item.Settings()).group(SpookyTime.GROUP)));
+	
+	public static Block ALCHEMICAL_BASIN_BLOCK = SpookyBlocks.register("alchemical_basin", new AlchemicalBasinBlock(FabricBlockSettings.of(Material.METAL).breakByHand(true).strength(0.5F, 0.5F).build()));
+	
+	public static final BlockEntityType<AlchemicalBasinBlockEntity> ALCHEMICAL_BASIN =
+			Registry.register(Registry.BLOCK_ENTITY, SpookyTime.id("alchemical_basin"), BlockEntityType.Builder.create(AlchemicalBasinBlockEntity::new, DOOM_HEART_BLOCK).build(null));
```

Comment:
The BE type should be applicable to the Basin block, why is it set to the Doom Heart block?

Reply:
'cuz it was wrong. :-). Thanks for spotting.  Can you tell I was in a hurry?